students/ghassan/lesson03/fruits.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# series1
+
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruits)
+new_fruit = input('Enter a new fruit: ')
+fruits.append(new_fruit)
+print(fruits)
+fruit_index = input('Enter a fruit index: ')
+print(fruits[int(fruit_index)-1])
+another_fruit = input('Enter another fruit: ')
+fruits = [another_fruit] + fruits
+print(fruits)
+yet_another_fruit = input('Enter yet another fruit: ')
+fruits.insert(0, yet_another_fruit)
+print(fruits)
+print([fruit for fruit in fruits if fruit.startswith('P')])
```

Comment:
nice use of comprehension here 👍 

always a good idea to use `lower()` on a string when doing comparison to be case not sensitive